client/components/review-field.js

```diff
@@ -53,6 +53,14 @@ class ReviewField extends React.Component {
       )
     }
 
+    if (this.props.type === 'holder' && this.props.licenceHolder) {
+      return (
+        <a href={`/e/${this.props.establishment.id}/people/${this.props.licenceHolder.id}`}>
+          {`${this.props.licenceHolder.firstName} ${this.props.licenceHolder.lastName}`}
+        </a>
```

Comment:
This needs to use a `<Link/>` component because the URL may be different depending on whether it's an internal or external service.

Reply:
`asl-projects` doesn't have `asl-components` as a dep (possibly because of drafting tool?).

I'll add a workaround that doesn't rely on it.

Reply:
@wheelsandcogs - have you ? @lennym -  I actually remember having had this conversation with @joefitter https://asl-marvell.slack.com/archives/C988GRGNN/p1566904267018900 , TLDR; use good old fashioned a  but pass it in as different for internal/external.

Reply:
The good news is that we have `@as/components` as a dependency now (or at least in one of @wheelsandcogs's PRs), so this can be fixed properly.

Reply:
https://github.com/UKHomeOffice/asl-projects/pull/367 << here